### ISSUE TITLE
resized-Image-Feature?

### ISSUE THREAD

**Issue by kangaroo72** [nextcloud/android#651]
Hi,

I've seen, the presentation from Tobias ...
[https://conf.nextcloud.com/conference/NextcloudConference2016/program/proposal/10](url)

Cool ...

@tobiasKaminsky ... my english isn't that perfect to understand everything. But am I wright, that I heard, you want to make it possible for a - let me say - "quick dia show over cache"? So we don't have to transfer the full picture-size?? Are these pictures also full screen??

And when is the feature coming? Do you have an average ETA?

Thanks, Tom

**Comment by tobiasKaminsky** [comment]
This is correct. It is currently in our Beta version, available on Fdroid or here 
https://github.com/nextcloud/android/blob/beta/apks/latest.apk if you might wanna try it.
We do not have an ETA, but I hope that we can integrate it soon as most of the code is already written by me.

**Comment by kangaroo72** [comment]
Hi Tobias.
I can confirm the installation of the app. And I tried the slideshow via cache. Amazing ...
Great feature - also the new gui. Very nice.
Are you on the meeting in Frankfurt this week?

**Comment by AndyScherzinger** [comment]
He can't make it, but I will be there and probably @eppfel.
Beware though that we changed the location to Die Zentrale Coworking :)

**Comment by AndyScherzinger** [comment]
And @juliushaertl

**Comment by kangaroo72** [comment]
I'll try - depends on my health ... same time - 19h?

**Comment by AndyScherzinger** [comment]
Yes 19:00 :)

**Comment by kangaroo72** [comment]
Just looked on the map ... the location before was nearly borderline for me (by train & bus from Wiesbaden). Berger Str. 175 is now to far for me :-( ...
Or someone coming by car from Wiesbaden / Mainz?

**Comment by AndyScherzinger** [comment]
Not that I know of but you can ask in the Meetup. Pity though but well next time then. The original, usual location is closed for renovation. :(
Might have to cancel March since @eppfel and I won't be able to make it.

**Comment by kangaroo72** [comment]
Possible to buy a NC-Shirt there? :-)))

**Comment by AndyScherzinger** [comment]
Unfortunately no, Felix and I got ours at the conference last year, but you can ping @jospoortvliet

**Comment by kangaroo72** [comment]
;-) thanks - I'm very interested in the meeting - maybe I'll make it possible .. we'll see ...
If so, are there signs where to go in?

**Comment by AndyScherzinger** [comment]
Unfortunately no, but it is the door to the left right in front of the main entrance to the Berger Cinema, but you can also ping me on twitter, same username :) 
(not wanting to post my cellphone number in public...)

**Comment by kangaroo72** [comment]
hmmm ... don't have twitter ...
there's a temporary mail, if you want to send it - thanks
l2420567@mvrht.com

**Comment by AndyScherzinger** [comment]
Sent you my number

**Comment by AndyScherzinger** [comment]
Bounced, or ping me on the forum. :)

**Comment by kangaroo72** [comment]
Just want to know ... is a duration of the meeting known?

**Comment by AndyScherzinger** [comment]
Usually 19:00 to around 22:00

**Comment by kangaroo72** [comment]
I think you're in middle of the meeting. It wasn't possible for me - maybe next time.
Have a nice evening!!

**Comment by kangaroo72** [comment]
@tobiasKaminsky the resized-feature isn't included in android-app 1.4.2 I think?

**Comment by AndyScherzinger** [comment]
@kangaroo72 no, the resized image feature is still work in progress, see further discussion in #69
